is-chrome.js

```diff
@@ -2,5 +2,5 @@
 
 module.exports = function isChrome(ua) {
   ua = ua || navigator.userAgent;
-  return ua.indexOf('Chrome') !== -1 || ua.indexOf('CriOS') !== -1;
+  return (ua.indexOf('Chrome') !== -1 || ua.indexOf('CriOS') !== -1) && !/SamsungBrowser/.test(ua);
```

Comment:
I think it would be nicer to use `isSamsungBrowser` here to reduce the duplication and prevent future issues.